pkg/plugins/runner/sorted_test.go

```diff
@@ -0,0 +1,62 @@
+package runner
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	"github.com/google/go-cmp/cmp"
+
+	"github.com/nornir-automation/gornir/pkg/gornir"
+	"github.com/nornir-automation/gornir/pkg/plugins/logger"
+)
+
+// TestSorted runs test func and verifies the hosts are executed
+// in alphabetical order by checking the results come in the right
+// order
+func TestSorted(t *testing.T) {
+	testCases := []struct {
+		name          string
+		expected      []string
+		sleepDuration time.Duration
+	}{
+		{
+			name:          "simple test",
```

Comment:
I would be more descriptive, `run in alphabetical order` or so. The future you will thank you if this test ever fails.